src/components/AppProvider/utilities/createAppProviderContext/tests/createAppProviderContext.test.tsx

```diff
@@ -107,11 +105,43 @@ describe('createAppProviderContext()', () => {
   });
 
   it('adds an app bridge hook to set clientInterface data', () => {
+    const set = jest.fn();
+    (createApp as jest.Mock<{}>).mockImplementation((args) => {
+      return {...args, hooks: {set}};
+    });
+
     const apiKey = '4p1k3y';
     createAppProviderContext({apiKey});
 
-    expect((createAppWrapper as jest.Mock<{}>).mock.calls[0][2]).toEqual([
-      hookMiddleware,
-    ]);
+    expect(set).toHaveBeenCalledWith(
+      LifecycleHook.DispatchAction,
+      setClientInterfaceHook,
+    );
+  });
+
+  it('does not add an app bridge hook if server side rendering', () => {
```

Comment:
I think we can remove this test since the assertion is covered here https://github.com/Shopify/polaris-react/blob/b53959d1d828dedfea9d6a92669f0889e98031ec/src/components/AppProvider/utilities/createAppProviderContext/tests/createAppProviderContext.test.tsx#L88